src/search/landmarks/landmark_count_heuristic.cc

```diff
@@ -54,11 +54,10 @@ LandmarkCountHeuristic::LandmarkCountHeuristic(const options::Options &opts)
 
     shared_ptr<LandmarkFactory> lm_graph_factory = opts.get<shared_ptr<LandmarkFactory>>("lm_factory");
     lgraph = lm_graph_factory->compute_lm_graph(task);
-    bool reasonable_orders = lm_graph_factory->use_reasonable_orders();
     lm_status_manager = utils::make_unique_ptr<LandmarkStatusManager>(*lgraph);
 
     if (admissible) {
```

Comment:
I decided to move the information whether reasonable orders are used or not to the Landmark Graph instead of the Factories, because it avoid propagating the information through all factories (analogous to supports_conditional_effects) and is a property of the graph anyway.

Reply:
I see Clemens' point, I think `has_reasonable_orders` feels more natural for the landmark graph than `uses`. But if you think that we will anyway move away from having this member at all by counting instead, then I'm fine with the current solution. Otherwise, I would prefer having a `has` method and ensuring that the user cannot use a factory which produces reasonable orders where this is not "safe" at another place.

Reply:
Yes. If this is the purpose, then this is clearly a property of the factory and not of the graph.